### DISCUSSION TITLE
Grid columns and media queries

### DISCUSSION THREAD

**Discussion by puzzle-it-nu** [ZTL-UwU/shadcn-docs-nuxt#57] [Q&A] [answered]
First of all: great work :+1: 

But I seemed to have run into a problem.
I would like to have a 4 column layout in a `::card-group` but it looks like the `class="grid grid-cols-4"` is overridden in the card-group component. (https://github.com/ZTL-UwU/shadcn-docs-nuxt/blob/aa0db3e90f5cb45e23205e109b26a194d06f4ca6/components/content/CardGroup.vue#L2)

I'm I missing something? besides overwriting `CardGroup.vue` 

```
::card-group{class="grid grid-cols-4"}
    ::card
    ---
    title: GB WegBeheer
    to: /wegbeheer
    target: _self
    ---

    ![](/images/software/gb_wegbeheer.jpg){class="size-full"}

    #footer
    Voor het beheren van wegen en verhardingen
    ::
// repeat the above card 3 more times and don't get a 4 column layout but a 2 column layout
::
```
Regards,

Joop

**Comment by ZTL-UwU** [comment, marked as answer]
Adding `md:grid-cols-4` will fix it.

```md
::card-group{class="grid-cols-4 md:grid-cols-4"}
  // ...
::

```

**Comment by puzzle-it-nu** [comment]
Thanks, that works indeed. I don't know how many things I've tried, thanks.

Joop
